Install lisp files in share/emacs/site-lisp if it exists.

*** send-pr/aclocal.m4	Fri Jul  7 13:52:09 1995
--- send-pr/aclocal.m4	Sat Nov 11 10:44:36 1995
***************
*** 36,43 ****
  if test -n "$with_lispdir"; then
    LISPDIR=${with_lispdir}
  else
!   for f in ${prefix-/usr/local}/lib/emacs/site-lisp \
! 	   ${prefix-/usr/local}/lib/emacs/lisp; do
      if test -d $f; then
        if test -n "$prefix"; then
  	LISPDIR=`echo $f | sed "s,^$prefix,"'$(prefix),'`
--- 36,45 ----
  if test -n "$with_lispdir"; then
    LISPDIR=${with_lispdir}
  else
!   for f in ${prefix-/usr/local}/share/emacs/site-lisp \
! 	   ${prefix-/usr/local}/share/emacs/lisp \
! 	   ${prefix-/usr/local}/lib/emacs/site-lisp \
! 	   ${prefix-/usr/local}/lib/emacs/lisp ; do
      if test -d $f; then
        if test -n "$prefix"; then
  	LISPDIR=`echo $f | sed "s,^$prefix,"'$(prefix),'`
***************
*** 50,56 ****
  fi
  if test -z "$LISPDIR"; then
  dnl # Change this default when Emacs 19 has been around for a while
!   LISPDIR='$(prefix)/lib/emacs/lisp'
  fi
  AC_MSG_RESULT(${LISPDIR})
  AC_SUBST(LISPDIR)dnl
--- 52,58 ----
  fi
  if test -z "$LISPDIR"; then
  dnl # Change this default when Emacs 19 has been around for a while
!   LISPDIR='$(prefix)/lib/emacs/site-lisp'
  fi
  AC_MSG_RESULT(${LISPDIR})
  AC_SUBST(LISPDIR)dnl
